Scripts/GameManagement/GameManager.gd

```diff
@@ -34,10 +33,16 @@ remotesync func initialize_players():
 		var player = picked_class_scene.instance()
 		player.set_name(str(p))
 		world.add_child(player)
-		player.position = Vector2(randi()%int(get_viewport().size.x), randi()%int(get_viewport().size.y))
+		player.position = random_spawnpoint()
 		players_info[p] = {"position": player.position, "body_rotation": player.get_node("Body").rotation}
 		get_tree().change_scene_to(world)
 		
+func random_spawnpoint():
+	var x_axis = [2054, 750, 3145, 4858, 8208, 11649, 2137, 7730, 5983, 3971, 6275, 9049, 11839, 10444, 9073, 6675, 12331]
+	var y_axis = [1510, 5000, 6606, 9790, 10184, 8987, 9001, 7470, 7184, 3498, 1179, 1604, 1826, 4098, 5777, 4295, 6470]
+	var number = randi()%17
```

Comment:
What if 2 players will spawn in the same place? You must create list for spawnpoints which are already used. And what if we would resize the coordinates list? I think instead of writing 17 we should use for example x_axis.size()